common/src/main/scala/com/gu/monitoring/products/RecurringContributionsMetrics.scala

```diff
@@ -0,0 +1,20 @@
+package com.gu.monitoring.products
+
+import com.gu.config.Configuration
+import com.gu.monitoring.ProductMetrics
+
+class RecurringContributionsMetrics(val paymentMethod: String,
```

Comment:
I think you could put `override` for these two `val`s so that it is clear that they are used in the parent class.  Otherwise they look unused